File: C++/numberofdigits.cpp
// 15) Program to count number of digits in an integer
#include<bits/stdc++.h>
using namespace std;

int main(){
    int num;
    cin>>num;

    int count = 0;
    while(num != 0){
        num = num/10;
        count++;
    }

    cout<<"Number of digits in entered number is: "<<count<<endl;
    return 0;
}

File: Python/number_of_digits.py
# Function to calculate the number of digits in the given input integer
def digits(val):
    if val.isnumeric:
        # negating the minus '-' sign
        if val[0] == '-':   
            return ("Number of digits in " + val + " is " + str(len(val)-1))
        else:
            return ("Number of digits in " + val + " is " + str(len(val)))
    else:
        return ("Invalid Input!")

#Prompting User for input
num = input("Enter an integer: ")

print(digits(num))
